packages/react-saasify/src/components/ServiceForm/ServiceInputWrapperWithLabel.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { ServiceInputWrapper } from './ServiceInputWrapper'
+import theme from 'lib/theme'
+import styles from './styles.module.css'
+
+export const ServiceInputWrapperWithLabel = ({
+  children,
+  propKey,
+  description
+}) => (
+  <ServiceInputWrapper>
+    <label>
+      <div className={theme(styles, 'service-form__input-label')}>
+        <span className={theme(styles, 'service-form__input-name')}>
+          {propKey} -{' '}
+        </span>
+
+        <span className={theme(styles, 'service-form__input-description')}>
+          {description}
```

Comment:
parameters won't always have a description. we should think of any of this type of metadata as optional albeit recommended, with any of our own internal launches & examples always having stuff like this polished.

Reply:
Will optionally render